lizmap/modules/lizmap/lib/Project/QgisProject.php

```diff
@@ -1268,6 +1268,7 @@ protected function readThemes($xml)
                     $themes[(string) $themeObj->name]['layers'][(string) $layerObj->id] = array(
                         'style' => (string) $layerObj->style,
                         'expanded' => (string) $layerObj->expanded,
+                        'visible' => isset($layerObj->visible) ? (string) $layerObj->visible : '1',
```

Comment:
@nboisteault  To avoid getting all layers, we probably can do something like that:
```php
                     // do not keep layer with visible != '1' if it is defined
                     if ( isset($layerObj->visible) && (string) $layerObj->visible != '1')
                        conitnue;
                    $themes[(string) $themeObj->name]['layers'][(string) $layerObj->id] = array(
                        'style' => (string) $layerObj->style,
                        'expanded' => (string) $layerObj->expanded,
                    );
```
And revert the JS changes.